src/Shared/Recruit.Vacancies.Client/Application/Validation/Fluent/ApplicationReviewValidator.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using FluentValidation;
+
+namespace Esfa.Recruit.Vacancies.Client.Application.Validation.Fluent
+{
+    public class ApplicationReviewValidator : AbstractValidator<ApplicationReview>
+    {
+        public const int CandidateFeedbackMaxLength = 200;
+
+        public const string OutcomeRequired = "You must select either successful or unsuccessful";

```

Comment:
As mentioned, I think we should put these on Confluence so the error message content can be reviewed and tracked. Error codes would also be good because of the other systems that will use the client package in future just like Vacancy

Reply:
👍 Will do.  I think we should review the error codes when we need them.